recipes/lerclib/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class LerclibConan(ConanFile):
+    name = "lerclib"
```

Comment:
```suggestion
    name = "lerc"
```

Use the original project name. Even Conda uses "lerc"

Reply:
ok, recipe renamed lerc (and also name in conanfile)